File: Phasegate/DayOne/Level 2/python/TEST/task_one.py
def move_zeros_to_end(numbers):
    result = []
    for index in range(len(numbers)):
        if numbers[index] != 0:
            result.append(numbers[index])
    for index in range(len(numbers)):
        if numbers[index] == 0:
            result.append(0)
    return result

File: Phasegate/DayOne/Level 2/java/TEST/TaskOne.java
public class TaskOne{
    public static int[] moveZerosToEnd(int[] numbers){
        int nonZeroCount = 0;
        for (int index = 0; index < numbers.length; index++){
            if (numbers[index] != 0){
                nonZeroCount++;
            }
        }
        int[] result = new int[numbers.length];
        int resultIndex = 0;
        for (int index = 0; index < numbers.length; index++){
            if (numbers[index] != 0){
                result[resultIndex] = numbers[index];
                resultIndex++;
            }
        }
        for (int index = resultIndex; index < numbers.length; index++){
            result[index] = 0;
        }

        return result;
    }
}
